packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 42.4.0 - February 22, 2018
+### Added
+#### salesforcedx-vscode-core
+
+* Include source links to apex classes on apex test failures ([PR #308](https://github.com/forcedotcom/salesforcedx-vscode/pull/308))
+
+#### salesforcedx-utils-vscode
```

Comment:
As a collaborator this is a nice feature but not sure if it should be added to the changelog.

Reply:
Yeah I would remove #295.